sales.html

```diff
@@ -15,7 +15,23 @@ <h1 class="header">Seattle Cookies</h1>
     <section>
       <h1 class="title">Cookies Sold Per Hour</h1>
     </section>
+    <form id = "new-location">
```

Comment:
@azizaalhouni @kaylynyuh multiple instances in this html file where the attribute and associated value do not directly touch the =.  For example, you should have id="new-location" vs id = "new-location"

while this still seemingly works, older browsers will break when attempting to render this